pydy/viz/static/js/dyviz/scene.js

```diff
@@ -246,7 +246,7 @@ DynamicsVisualizer.Scene = Object.extend(DynamicsVisualizer, {
             case "Circle":
                 var geometry = new THREE.CylinderGeometry(object.radius,
                                                           object.radius,
-                                                          0.5,100);
+                                                          object.radius/25,100);
```

Comment:
this is the line you could enter zero or a higher divisor if you want a thinner disc shape.